react/features/video-menu/actions.web.ts

```diff
@@ -15,3 +17,20 @@ export function renderConnectionStatus(showConnectionInfo: boolean) {
         showConnectionInfo
     };
 }
+
+/**
+ * Notify the first participant in the speaker queue to prepare to speak.
+ *
+ * @returns {Function} A function that dispatches a notification action.
+ */
+export function notifyToSpeak() {
```

Comment:
Note for the next PR: don't put this in `actions.web` since we'll want to reuse it for mobile too.